src/content/getting-started/quickstart/external/_index.md

```diff
@@ -0,0 +1,429 @@
+
+    | Host      | IP              | Remarks            |
+    |-----------|-----------------|--------------------|
+    | test-vm   | 192.168.122.142 | Linux is installed |
+
+    In above case, all of them are deployed inside 192.168.122.0/24 segment.
+    However, it is only required that cluster-a and test-vm are in the same segment.
+    Other clusters, cluster-b and any additional clusters, can be deployed in different segments or even in any other networks in the internet. Also, these clusters can be multi-node cluster.
+
+    On the other hand, subnets of non-cluster hosts should be distinguished from those of all the clusters to easily specify the external network CIDR.
+    In above case, cluster-a and cluster-b belongs to 192.168.122.0/25 network and test-vm belongs to 192.168.122.128/25 network.
+    Therefore, the external network CIDR for this configuration is 192.168.122.128/25.
+    In test environment for just one host, we will be able to specify external network CIDR, like 192.168.122.142/32, however design of the subnet needs to be considered when more hosts are added.
+
+2. Choose the Pod CIDR and the Service CIDR for k8s clusters and deply them.
+
+    In this guide, we will use the following CIDRs:
+
+    | Cluster   | Pod CIDR     | Service CIDR |
+    |-----------|--------------|--------------|
+    | cluster-a |10.42.0.0/24  |10.43.0.0/16  |
+    | cluster-b |10.42.0.0/24  |10.43.0.0/16  |
+
+    Note that we will use globalnet in this guide, therefore overlapping CIDR is allowed.
+    One of the easiest way to create this environment will be to deploy two K3s clusters by the steps described [here](https://submariner.io/getting-started/quickstart/k3s/) until "Deploy cluster-b on node-b", with modifying deploy commands to just `curl -sfL https://get.k3s.io | sh -` to use default CIDR.
+
+### Set up submariner
+
+#### Ensure kubeconfig files
+
+Ensure that kubeconfig file for cluster-a exists as kubeconfig.cluster-a and kubeconfig file for cluster-b exists as kubeconfig.cluster-b on node of cluster-a. File names can be different, but rest of this guide assumes these files are named as such.
+
+#### Install `subctl` on node of cluster-a
+
+{{% subctl-install %}}
+
+#### Use cluster-a as the Broker with globalnet enabled
+
+On node of cluster-a, run:
+
+```bash
+subctl deploy-broker --kubeconfig kubeconfig.cluster-a  --globalnet
+```
+
+#### Join cluster-a to the Broker with external CIDR added as clustercidr
+
+Carefully review the `CLUSTER_CIDR` and `EXTERNAL_CIDR` and run:
+
+```bash
+CLUSTER_CIDR=10.42.0.0/24
+EXTERNAL_CIDR=192.168.122.128/25
+subctl join --kubeconfig kubeconfig.cluster-a broker-info.subm --clusterid cluster-a --natt=false --clustercidr=${CLUSTER_CIDR},${EXTERNAL_CIDR}
+```
+
+#### Join cluster-b to the Broker
+
+```bash
+subctl join --kubeconfig kubeconfig.cluster-b broker-info.subm --clusterid cluster-b --natt=false
+```
+
+#### Deploy dns server on cluster-a for non-cluster hosts
```

Comment:
Created a separate issue to discuss a better way for this, [here](https://github.com/submariner-io/lighthouse/issues/600).